bundles/org.openhab.binding.nuvo/src/main/java/org/openhab/binding/nuvo/internal/handler/NuvoHandler.java

```diff
@@ -184,6 +192,11 @@ public void initialize() {
             return;
         }
 
+        this.isMps4 = port.intValue() == MPS4_PORT;
```

Comment:
I found a small issue... 'port' needs to be checked for null. When port is left blank in the configuration it throws an NPE here.
```suggestion
        this.isMps4 = (port != null && port.intValue() == MPS4_PORT);
```